docs/digger/installation/installer-variables.asciidoc

```diff
@@ -0,0 +1,295 @@
+== Variables
+
```

Comment:
@wei-lee do we expect users to edit these values? if so, how? (inventory file?)

Reply:
@finp yes the user can override values from the inventory file. This is outline in the run-digger-installer.asciidoc file and I will point to this file form there.